src/shared/db.py

```diff
@@ -176,12 +178,22 @@ def remove_from_db(self, uid: str) -> bool:
 # created in DbAccount
 class SubHubDeletedAccountModel(Model):
     user_id = UnicodeAttribute(hash_key=True)
-    cust_id = UnicodeAttribute(null=True)
+    cust_id = UnicodeAttribute(range_key=True)
     origin_system = UnicodeAttribute()
     customer_status = UnicodeAttribute()
+    subscription_info = ListAttribute()
 
 
 def _create_deleted_account_model(table_name_, region_, host_) -> Any:
+    class CustomerIndex(GlobalSecondaryIndex):
+        class Meta:
+            index_name = "cust-index"
+            read_capacity_units = 2
```

Comment:
Again as a follow up, we should discuss this in a follow up around the database model.  It shouldn't though hold up this pull request.

Reply:
Story?

Reply:
https://jira.mozilla.com/browse/COPS-784